client/__main__.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
+
+import os
+import argparse
+
+from logbook import Logger, INFO, DEBUG, NullHandler, NestedSetup
+from logbook.queues import ZeroMQHandler, ZeroMQSubscriber
+from logbook.more import ColorizedStderrHandler
+
+from onitu.utils import get_available_drivers, get_logs_uri
+
+os.environ['ONITU_CLIENT'] = 'client'
+
+logger = None
+
+
+def get_logs_dispatcher(uri=None, debug=False):
+    """Configure the dispatcher that will print the logs received
+    on the ZeroMQ channel.
+    """
+    handlers = []
+
+    if not debug:
+        handlers.append(NullHandler(level=DEBUG))
+
+    handlers.append(ColorizedStderrHandler(level=INFO))
+
+    if not uri:
+        uri = get_logs_uri('client')
+
+    subscriber = ZeroMQSubscriber(uri, multi=True)
+    return uri, subscriber.dispatch_in_background(setup=NestedSetup(handlers))
+
+
+def get_setup(setup_file):
```

Comment:
Also the version from `onitu.__main__` should be the one moved to the utils, as it has changed.